src/howTos/dev/services.md

```diff
@@ -111,20 +105,19 @@ Each time you make modifications to your service, you must update the app on the
 cozy-stack apps update <app_name>
 ```
 
-
 ## Execution
 
 The service will be run each time the trigger condition is met, e.g. a bank operation.
-However, you can force its execution thanks to the `cozy-run-dev` CLI.
+However, you can force its execution thanks to the `cozy-run-dev` CLI, which can be useuful for developement.
```

Comment:
Below, its written cozy-konnector-dev ? For a service, it should be cozy-run-dev right ?

s/useuful/useful

Reply:
On a eu des soucis avec `cozy-run-dev` qui ne comprenait pas que le fichier buildé devait être exécuté par node et non dans un shell. Il fallait de plus faire un chmod +x sur le fichier buildé

Reply:
I changed it for `cozy-konnector-dev`. Not sure why it didn't work with `cozy-run-dev`, though